modules/build/src/main/scala/scala/build/preprocessing/mdsandbox/SnippetPackager.scala

```diff
@@ -0,0 +1,147 @@
+package scala.build.preprocessing.mdsandbox
+
+import MarkdownSnippet.Fence
+
+class SnippetPackager(val fileName: String, val snippets: Seq[Fence]) {
+  def this(fileName: String, content: String) = this(fileName, MarkdownSnippet.findFences(content))
```

Comment:
it'd make sense to make this constructor into an `apply` method on the companion object